ppa/editorial/templates/editorial/snippets/attribution.html

```diff
@@ -0,0 +1,10 @@
+{# Snippet to include authors and date attribution to editorial posts #}
+{% for block in page.authors %}
+    {% if block.value.url and add_link %}
+        <a href="{{ block.value.url }}" target="_blank">
+    {% endif %}
+        By {{ block.value.name }}
+        {% if block.value.url and add_link %}</a>{% endif %}
+        {% if not forloop.last %}, {% endif %}
```

Comment:
The newline is rendering as a space between author name and comma when this displays.

Reply:
Fixing, too, obviously.